.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+blank_issues_enabled: false
+contact_links:
+  - name: Community Support
+    url: https://github.com/apache/fury/discussions
+    about: Please ask and answer questions here.
+  - name: Security Bug Bounty
```

Comment:
Security report should not be here. Reporter should follow https://github.com/apache/.github/blob/main/.github/SECURITY.md

Reply:
got it

Reply:
done